test/plugin/scenarios/resttemplate-4.x-scenario/config/expectedData.yaml

```diff
@@ -19,11 +19,29 @@ registryItems:
   instances:
   - {resttemplate-4.x-scenario: 1}
   operationNames:
-  - resttemplate-4.x-scenario: [/resttemplate/case/resttemplate, /resttemplate/back]
+  - resttemplate-4.x-scenario: [/resttemplate/case/healthcheck, /resttemplate/case/resttemplate, /resttemplate/back]
```

Comment:
`/resttemplate/case/healthcheck` should not be added into here.

Reply:
okay